test/IRGen/conditional-dead-strip-reflection.swift

```diff
@@ -0,0 +1,31 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-build-swift -Xfrontend -conditional-runtime-records %s -emit-ir -o %t/main.ll
+// RUN: %target-clang %t/main.ll -isysroot %sdk -L%swift_obj_root/lib/swift/%target-sdk-name -flto -o %t/main
+// RUN: %target-swift-reflection-dump -binary-filename %t/main | %FileCheck %s
+
+// FIXME(mracek): More work needed to get this to work on non-Apple platforms.
+// REQUIRES: VENDOR=apple
```

Comment:
What work is needed for non-Apple targets?

Reply:
I haven't fully investigated yet (it's on my list) but it's due to us using llvm.compiler.used instead of llvm.used.